j2cl/lit/src/tokens/shell-tokens.css

```
@@ -0,0 +1,257 @@
+  --shell-color-surface-soft: #f1f4f5;
+  --shell-color-surface-rail: #ebeeef;
+  --shell-color-text-primary: #181c1d;
+  --shell-color-text-muted: #3e494a;
+  --shell-color-divider-subtle: #bdc8ca;
+  --shell-color-accent-brand: #00636e;
+  --shell-color-accent-focus: #087e8b;
+  --shell-space-inline-default: 12px;
+  --shell-space-inset-panel: 18px;
+  --shell-shell-radius: 20px;
+  --shell-shell-shadow: 0 24px 40px rgba(24, 28, 29, 0.06);
+  --shell-font-ui: "Manrope", "Public Sans", -apple-system, BlinkMacSystemFont, "Segoe UI",
```

Stylelint (17.7.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)